### PR TITLE
Organize data.swift

### PR DESCRIPTION
**Pull request by venetucci**
Organize colors in data.swift
Add dashed line extension for UIView
Initial set up for first proximity view controller

image

### CODE REVIEW

File: overlay/ProximityOneViewController.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+//  ProximityOneViewController.swift
+//  overlay
+//
+//  Created by Michelle Harvey on 3/16/16.
+//  Copyright © 2016 overlay. All rights reserved.
+//
+
+import UIKit
+
+class ProximityOneViewController: UIViewController {
```
Comment (emmaguo) [inline overlay/ProximityOneViewController.swift:11]: Move this file to `View Controllers` folder.

File: overlay/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="9532" systemVersion="15C50" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" initialViewController="BYZ-38-t0r">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="9532" systemVersion="14F27" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" initialViewController="nkz-hu-AKw">
```
Comment (emmaguo) [inline overlay/Base.lproj/Main.storyboard]: We should keep `homeViewController` as the initial view controller.

File: overlay/ProximityOneViewController.swift (hunk 1 above)
Comment (venetucci) [inline overlay/ProximityOneViewController.swift:11]: I think it is already? 

image

File: overlay/Base.lproj/Main.storyboard (hunk 2 above)
Comment (venetucci) [inline overlay/Base.lproj/Main.storyboard]: Oh, does this matter? I guess it's a conflict? I can push new changes...

**Comment by venetucci** [comment]
@emmaguo updated the initial view controller, and added a tiny bit of code to `ProximityOneViewController`

File: overlay/ProximityOneViewController.swift [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        
+        zoneOneView.addInactiveDashedBorder()
+        zoneTwoView.addInactiveDashedBorder()
+        
+        unitAOne.backgroundColor = proximityColor
+        unitATwo.backgroundColor = proximityColor
+        unitAThree.backgroundColor = proximityColor
+        unitBOne.backgroundColor = alignmentColor
+        unitBTwo.backgroundColor = alignmentColor
+        unitBThree.backgroundColor = alignmentColor
```
Comment (venetucci) [inline overlay/ProximityOneViewController.swift:37]: @emmaguo I added these colors

**Comment by emmaguo** [comment]
:+1: 
:tada:

**Pull request merged**